### PR TITLE
Added combineMeshes stage

### PR DESCRIPTION
**Pull request by leerichard42**
This is a branch off the combinePrimitives stage, and I made a small fix in the jasmine describe header for the combinePrimitives test in addition to the combineMeshes stage.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Do you want to call `combineMeshes` from index.js for the example pipeline?

**Comment by pjcozzi** [comment]
Part of #43.

File: lib/combineMeshes.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+module.exports = combineMeshes;
+
+//Combines meshes in a node.
+function combineMeshes(gltf) {
```
Comment (pjcozzi) [inline lib/combineMeshes.js]: This needs to run before the `combinePrimitives` stage, right?  This creates just one mesh with a big array with all the primitives, then it is expected that `combinePrimitives` will combine primitives.

I think it is worth adding a comment here.

**Comment by pjcozzi** [comment]
Great start, @leerichard42!

Can you also add test case for the case @tfili mentioned in https://github.com/AnalyticalGraphicsInc/gltf-pipeline/issues/43#issuecomment-209470582?

This would be a bit more of an integration test where the test would have both the combine mesh and primitives stages.

File: lib/combineMeshes.js (hunk 1 above)
Comment (leerichard42) [inline lib/combineMeshes.js]: Yes, I will add this comment.

**Comment by leerichard42** [comment]
I thought that the transforms test case was for the combine nodes stage? Because wouldn't combining the nodes, rather than the meshes, affect the data, because the nodes with different transforms would be merged into one node with a single transform?

**Comment by pjcozzi** [comment]
> I thought that the transforms test case was for the combine nodes stage? Because wouldn't combining the nodes, rather than the meshes, affect the data, because the nodes with different transforms would be merged into one node with a single transform?

Ah, yes, you are right.  I was confusing the glTF and COLLADA terminology.

**Comment by pjcozzi** [comment]
Great work as always!

**Pull request merged**